pom.xml

```diff
@@ -1,69 +1,71 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
 
-    <artifactId>motech-ui-testing</artifactId>
-    <packaging>pom</packaging>
-    <version>0.27.1-SNAPSHOT</version>
+	<artifactId>motech-ui-testing</artifactId>
+	<packaging>pom</packaging>
+	<version>0.27.1-SNAPSHOT</version>
 
-    <name>MOTECH UI Functional Testing</name>
+	<name>MOTECH UI Functional Testing</name>
 
-    <modules>
-        <module>framework</module>
-        <module>tests</module>
-    </modules>
+	<modules>
+		<module>framework</module>
+		<module>tests</module>
+	</modules>
 
-    <properties>
-        <motech.version>0.27.5</motech.version>
-        <motech.dir>${user.home}/.motech</motech.dir>
-        <motech.sql.dbtype>mysql</motech.sql.dbtype>
+	<properties>
+		<motech.version>0.27.6</motech.version>
+		<motech.dir>${user.home}/.motech</motech.dir>
+		<motech.sql.dbtype>mysql</motech.sql.dbtype>
 
-        <scm.connection>scm:git:https://github.com/motech/UITestFramework</scm.connection>
-        <scm.developerConnection>scm:git:https://github.com/motech/UITestFramework</scm.developerConnection>
-        <scm.url>https://github.com/motech/UITestFramework</scm.url>
-    </properties>
+		<scm.connection>scm:git:https://github.com/motech/UITestFramework</scm.connection>
+		<scm.developerConnection>scm:git:https://github.com/motech/UITestFramework</scm.developerConnection>
+		<scm.url>https://github.com/motech/UITestFramework</scm.url>
+	</properties>
 
-    <scm>
-        <connection>${scm.connection}</connection>
-        <developerConnection>${scm.developerConnection}</developerConnection>
-        <url>${scm.url}</url>
-      <tag>HEAD</tag>
-  </scm>
+	<scm>
+		<connection>${scm.connection}</connection>
+		<developerConnection>${scm.developerConnection}</developerConnection>
+		<url>${scm.url}</url>
+		<tag>HEAD</tag>
+	</scm>
 
-    <parent>
-        <groupId>org.motechproject</groupId>
-        <artifactId>motech</artifactId>
-        <version>0.27.5</version>
-    </parent>
+	<parent>
+		<groupId>org.motechproject</groupId>
+		<artifactId>motech</artifactId>
+		<version>0.27.5</version>
```

Comment:
@rubenmartin you need to change the version here, I think you can use ${motech.version}